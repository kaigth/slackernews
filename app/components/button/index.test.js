import React from 'react';
import Button from './index';
import renderer from 'react-test-renderer';

it('renders the Filter correctly', () => {
  const tree = renderer
    .create(<Button />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});