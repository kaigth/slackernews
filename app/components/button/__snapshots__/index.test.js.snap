// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Filter correctly 1`] = `
<button
  className="button undefined"
  onClick={undefined}
  onKeyPress={[Function]}
  tabIndex="0"
/>
`;
